app/views/sprints/index.tsx

```diff
@@ -31,9 +31,16 @@ export default function ({
           {sprints.map((sprint) => (
             <Card
               key={sprint.id}
-              title={sprint.title}
+              title={<div>{sprint.title} <small>({l.dateRange(sprint.sprintFrom, sprint.sprintUntil)})</small></div>}
               icon="🏃"
-              subtitle={l.dateRange(sprint.sprintFrom, sprint.sprintUntil)}
+              subtitle={<Stack line='mobile'>
+                <div>🔢 {sprint.finishedStorypoints} pts</div>
+                <div>🔢 {l.singleDigitNumber(sprint.finishedStorypointsPerDay)} pts/day</div>
+                <div>⭐️ {l.singleDigitNumber(sprint.averageRating)}/5</div>
+                <div>💻 {sprint.totalWorkingDays} days</div>
+                {sprint.turnoverPerStorypoint && <div>💸 {l.currency(sprint.turnoverPerStorypoint)} per point</div>}
```

Comment:
@JensRavens at least in dev, turnover is returning 0 which makes the shortcircuiting return a 0 instead of empty and it's being rendered. Should it return null or do I change the condition here to check for numeric value?

Reply:
That's probably due to missing data in the cost rates. This value is always there, so it makes sense to make it non-null.